src/components/DashboardHome/Footer.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
Do we need this component, or could we leverage the one Alex developed at inventory/nav.js? (Or use yours instead...I just don't want to see two components with the same code in our codebase :))

Reply:
Understood. I believe this is resolved as Alex says his footer version has a bug. Also his footer wasn't a standalone component, so I'm keeping mine. Hope this works!

Reply:
Sure. Happy with that.